modules/SofaGeneralAnimationLoop/src/SofaGeneralAnimationLoop/initSofaGeneralAnimationLoop.cpp

```diff
@@ -34,9 +34,10 @@ void initSofaGeneralAnimationLoop()
     static bool first = true;
     if (first)
     {
-        // msg_deprecated("SofaGeneralAnimationLoop") << "SofaGeneralAnimationLoop is deprecated. It will be removed at v23.06. You may use Sofa.Component.Mapping.MappedMatrix instead.";
+        // msg_deprecated("SofaGeneralAnimationLoop") << "SofaGeneralAnimationLoop is deprecated. It will be removed at v23.06. You may use Sofa.Component.Mapping.MappedMatrix and Sofa.Component.AnimationLoop instead.";
```

Comment:
MappedMatrix?

Reply:
https://github.com/sofa-framework/sofa/pull/2635

```
MappedMatrix: components which are not mapping? but helps to map matrices (stiffness and stuff)
-> Ad-hoc module which is only used to patch up matrices in SOFA, will disappear once matrix assembly will be pushed
```

Reply:
just to be sure I got it. This msg_deprecated will be uncommented for the 22.12, and the compat layer will be removed in 23.06 ?

Reply:
I will uncomment the msg_warning before the v22.06 release